packages/commonwealth/server/controllers/server_admin_methods/get_stats.ts

```diff
@@ -85,27 +82,13 @@ export async function __getStats(
       `SELECT id FROM "Communities" WHERE created_at >= NOW() - INTERVAL '30 days'`,
       { type: QueryTypes.SELECT },
     ),
-    this.models.Comment.count({
-      where: whereCommunityId,
-    }),
-    this.models.Thread.count({
-      where: whereCommunityId,
-    }),
-    this.models.Reaction.count({
-      where: whereCommunityId,
-    }),
-    this.models.Vote.count({
-      where: whereCommunityId,
-    }),
-    this.models.Poll.count({
-      where: whereCommunityId,
-    }),
-    this.models.Address.count({
-      where: whereCommunityId,
-    }),
-    this.models.Group.count({
-      where: whereCommunityId,
-    }),
+    this.models.Comment.count(countQuery),
```

Comment:
We should optimize and use a `UNION ALL` query here.

Reply:
Updated it to use a union